modules/administration-guide/pages/finding-the-fqdn-of-your-organizations-instance.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: PROCEDURE
+:description: Finding the FQDN of your organization's instance of {prod-short}
+:keywords: administration-guide, FQDN, URL
+:navtitle: Finding the FQDN of your organization's instance of {prod-short}
+//:page-aliases:
+
+[id="finding-the-fqdn-of-your-organizations-instance"]
+= Finding the FQDN of your organization's instance of {prod-short}
```

Comment:
Does it make sense to explain the abbreviation in this doc? E.g. 

title: "Finding the fully-qualified domain name of your organization's instance of {prod-short}"

first sentence: "You can get the fully-qualified domain name (FQDN) of your organization's instance..."

Reply:
agree, let's make it clear what FQDN is

Reply:
Thank you both. I will do this.